libraries/botbuilder-core/tests/turnContext.test.js

```diff
@@ -305,7 +305,7 @@ describe('TurnContext', function () {
         });
     });
 
-    it('should round trip a conversation reference using getConversationReference() and applyConversationRefernce().', function () {
+    it.only('should round trip a conversation reference using getConversationReference() and applyConversationRefernce().', function () {
```

Comment:
What is this for?

Reply:
I added it just to run that test, but I forgot to remove it.